assets/vcfanno.toml

```diff
@@ -0,0 +1,5 @@
+[[annotation]]
+file="annotsv_annotated.vcf.gz"
+fields=["AnnotSV_ID"]
```

Comment:
So for `fields` I would suggest to use only fields that are present in the `full`annotation mode (see comment above). 
For now let's include these: 
```
AnnotSV ID
Samples_ID
FORMAT
Gene_name
Gene_count
Cosmic_ID
RE_gene
CytoBand
OMIM_ID
GnomAD_pLi
ExAC_pLi
ACMG_class
AnnotSV_ranking_score
AnnotSV_ranking_criteria
```

Reply:
Thanks! Any suggestions for the names of these? Or should I just keep the original names?

Reply:
Allright, the `FORMAT` isn't part of the `INFO` in the VCF so I don't think it's really possible to add these. I think AnnotSV doesn't change anything there though, so the VEP VCF should have the same `FORMAT`